man/scrot.txt

```diff
@@ -188,6 +192,13 @@ NOTE FORMAT
     $ scrot -n "-f '/usr/share/fonts/TTF/DroidSans-Bold/40' -x 10
             -y 20 -c 255,0,0,255 -t 'Hi'"
 
+COMPRESSION QUALITY
+
+For lossless formats (e.g PNG), the quality options is ignored. For lossy
+formats where the quality and compression are tied together (e.g JPEG),
+compression will be ignored. And for image formats where quality and
+compression can be independently set (e.g WebP, JXL), both flags are respected.
```

Comment:
NOTE: for formats that support both lossless and lossy encoding, quality of 100 typically implies lossless.

However I didn't mention it in the manpage because current imlib2 version doesn't support lossless webp (it's added in git, so will be supported next release).